lib/vmpooler/pool_manager.rb

```diff
@@ -1,5 +1,12 @@
 module Vmpooler
   class PoolManager
+
+    # Defaults
+    # port number to verify pending VM is ready
+    DEFAULT_PORT_NO      = 22
+    # timeout, in seconds, when connecting to DEFAULT_PORT_NO
+    DEFAULT_PORT_TIMEOUT = 5
+
```

Comment:
There are some defaults defined at https://github.com/puppetlabs/vmpooler/blob/master/lib/vmpooler.rb#L27-L34; could this be moved to the same spot?